setup.cfg

```diff
@@ -1,37 +1,34 @@
 [metadata]
-package_name = wfc3tools
+name = wfc3tools
 description = Python Tools for HST WFC3 Data
-long_description = A package which contains wrappers for the calwf3 reduction scripts and tools to aid data analysis
+long_description = A package which contains wrappers for the calwf3 reduction scripts and tools to aid data analysis.
 author = STScI
 author_email = help@stsci.edu
 license = BSD
-edit_on_github = False
 github_project = spacetelescope/wfc3tools
-description-file = README.rst
+description_file = README.rst
 url = http://wfc3tools.readthedocs.io/
 homepage = http:://wfc3tools.readthedocs.io/
 project_urls =
     Bug Reports = https://github.com/spacetelescope/wfc3tools/issues/
     Source = https://github.com/spacetelescope/wfc3tools/
     Help = https://hsthelp.stsci.edu
+classifiers=
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Scientific/Engineering :: Astronomy',
+    'Topic :: Software Development :: Libraries :: Python Modules'
 
-[build-sphinx]
-source-dir = docs
-build-dir = docs
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
-[tool:pytest]
-minversion = 2.2
-norecursedirs = build docs/_build relic
-
-[entry_points]
-
-[bdist_wheel]
-# This flag says that the code is written to work on both Python 2 and Python
-# 3. If at all possible, it is good practice to do this. If you cannot, you
-# will need to generate wheels for each Python version that you support.
-universal=1
+[options]
```

Comment:
```suggestion
[options]
packages = find:
```

Reply:
oh does this replace find_packages() in setup.py? im confused about what belongs in this file vs that file, i had it in .py because thats what i've seen elsewhere.

Reply:
Yes, it is a different way to say the same thing.